test/integration/test_integration.py

```diff
@@ -0,0 +1,168 @@
+from unittest import mock, TestCase
+import argparse
+
+import fhirpipe
+import fhirpipe.cli.run as run
+from fhirpipe.load.fhirstore import get_fhirstore, get_mongo_client
+
+
+### Run from mapping file ###
+args = argparse.Namespace(
+    chunksize=None,
+    config="config-test.yml",
+    mapping="test/fixtures/mimic_mapping.json",
+    multiprocessing=False,
+    reset_store=True,
+    resources=None,
+    source=None,
+)
+
+
+@mock.patch("fhirpipe.cli.run.parse_args", return_value=args)
+def test_run_gql(_):
+    run.run()
+    assert_result_as_expected()
+
+
+"""
+### Run with graphQL queries ###
+args = argparse.Namespace(
+    chunksize=None,
+    config="config-test.yml",
+    mapping="mapping-file.json",
+    multiprocessing=False,
+    reset_store=True,
+    resources=None,
+    source=None,
+)
+
+@mock.patch("fhirpipe.cli.run.parse_args", return_value=args)
+def test_run_from_file(_):
```

Comment:
why is this commented out ?

Reply:
The function names were swapped out with test_run_gql.
But I commented this out because I currently don't have a way to init a Pyrog docker to test if querying the server to get the mapping works correctly

Reply:
The function names were swapped out with test_run_file.
But I commented this out because I currently don't have a way to init a Pyrog docker to test if querying the server to get the mapping and then transforming the data works as expected

Reply:
alright :+1: